src/AuthenticationServices/PublicPrivateKeyAuthentication.cs

```diff
@@ -0,0 +1,34 @@
+//
+// PublicPrivateKeyAuthentication.cs
+//
+// Authors:
+//	TJ Lambert <antlambe@microsoft.com>
+//
+// Copyright 2021 Microsoft Corporation
+//
+
+#if !TVOS && !WATCHOS
+
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+#nullable enable
+
+namespace AuthenticationServices {
+
+    // authenticationservices.cs contain the following attributes for this class
+    // [NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	public static partial class PublicPrivateKeyAuthentication {
+        [DllImport ("__Internal")]
```

Comment:
There is a Constants static class that contains all the variables needed to do the DllImport. You can see how to do it doing a git grep in the src directory
```bash
git grep DllImport
```
Which returns:
```
ARKit/ARSkeleton.cs:            [DllImport (Constants.ARKitLibrary)]
AVFoundation/AVSampleBufferExtensions.cs:        [DllImport (Constants.AVFoundationLibrary)]
AVFoundation/AVTypes.cs:                [DllImport (Constants.AVFoundationLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary,EntryPoint="vImageConvolveWithBias_ARGB8888")]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary,EntryPoint="vImageConvolveWithBias_ARGBFFFF")]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
Accelerate/vImageTypes.cs:              [DllImport (Constants.AccelerateImageLibrary)]
```
Which shows the you should be using Constants.AuthenticationServicesLibrary

Reply:
@rachelkang you were right! Thanks Manuel for showing how to find this!